Projects/ChangeTier/Sources/Navigation/ChangeTierNavigation.swift

```diff
@@ -0,0 +1,88 @@
+import Foundation
+import SwiftUI
+import hCore
+import hCoreUI
+import hGraphQL
+
+public class ChangeTierNavigationViewModel: ObservableObject {
+    @Published public var isEditTierPresented = false
+    @Published public var isEditDeductiblePresented = false
+    @Published public var isCompareTiersPresented = false
+    @Published public var isInsurableLimitPresented: InsurableLimits?
+    @Published public var isTierLockedInfoViewPresented = false
+
+    @StateObject var router = Router()
+    var vm = ChangeTierViewModel()
```

Comment:
Can we also init ChangeTierViewModel with proper contractId and changeTierSource? 
We are always going to have this 2 inputs and would be good to have it from the scratch